src/quest_manager/models.py

```diff
@@ -975,3 +989,142 @@ def get_minutes_to_complete(self):
 
         minutes = (self.first_time_completed - self.timestamp).total_seconds() / 60
         return minutes
+
+
+class QuestionType(models.TextChoices):
+    """Identifies what types of questions can be created.
+
+    This is used in the views and forms to determine what type of question is being created, and to show/hide specific
+    fields. For example, with a short answer question, the solution_file field is hidden, but with a file upload question,
+    the solution_file field is shown.
+    """
+
+    SHORT_ANSWER = "short_answer", "Short Answer"
+    LONG_ANSWER = "long_answer", "Long Answer"
+    FILE_UPLOAD = "file_upload", "File Upload"
+
+
+class Question(models.Model):
+    """Used for adding additional submission fields to Quests.
+
+    This model is used for all types of questions, including multiple choice, short answer, file upload, etc.
```

Comment:
Good for now, but once you get this working, please add a description how it handles different questions types.  Maybe a list of relevant fields for each type or similar info.

Reply:
Sure! I'll write down that I have to do that.